src/sync.rs

```diff
@@ -25,14 +26,25 @@ pub struct PushErr {
     pub message: String,
 }
 
-#[derive(Debug, Eq, PartialEq, Clone)]
+#[derive(Debug, Eq, PartialEq, Clone, Deserialize)]
+#[serde(rename_all = "lowercase")]
 pub enum PullMode {
     /// Serial, after remote command execution.
     Serial,
 
     /// Parallel to remote command execution.
     /// First parameter is pause between pulls.
-    Parallel(Duration),
```

Comment:
Duration as a part of `PullMode` was removed due to the config file currently not supporting passing a value for it.  Is this okay to keep or should we figure out a syntax for passing the duration?

Reply:
Well, we kind of need it, but can def be done in a separate PR — just open an issue if you decide to do it separately :)

Reply:
I think we should do it separately to be able to discuss what the config would look like, documented in #284 👍